### PR TITLE
Updated rocSOLVER APIs documentation

### PR DESCRIPTION
**Pull request by jibrealkhan**
Description: Updated the API documentation according to the new notations.
[SWDEV-402069]

### CODE REVIEW

**Review by jzuniga-amd — COMMENTED** [review]
I opened a PR to your branch with some minor fixes. Please have a look. 
Other than that, it looks like the new format style fixes most of the issues that we had... so, the introduction of the new notation across all sections of the docs is a solid step in the right direction.

(We will still have to fix some things like standardizing the style of parameter names that appear in math formulas, the use of new lines and spaces in the parameter descriptions, etc., but this is detailed work that we can do little by little in the future as part of a defect  ticket)

**Comment by tfalders** [comment]
Still making my way through the changes, though I have a few comments for what I've seen so far. Unfortunately, I can't load the rocsolver-functions.h diff in my browser, so I can't link to the lines.

1. Is there a specific reason that `incp` was changed back to `incx` for LASWP? I'd prefer it to stay as `incp` if not.
2. It looks like the documentations for LAUUM, BDSVDX, GESVDJ, GESVDX, GEBLTTRF, GEBLTTRS, and the CSRRF functions still have newlines.
3. The batch indices for GEBLTTRF and GEBLTTRS might need another pass. I see some j's where there should be l's.
4. There are some isolated batch indices in GETRF_NPVT_BATCHED, GELQF_STRIDED_BATCHED, POSV_STRIDED_BATCHED, GESVDJ_BATCHED, GESVDJ_STRIDED_BATCHED, GESVDX_STRIDED_BATCHED, SYTRD_BATCHED, HETRD_BATCHED, HEEVD_STRIDED_BATCHED, and GETRI_OUTOFPLACE_STRIDED_BATCHED that need to be changed from j to l. (Hint: search for `_j` and `[j`)

File: library/include/rocsolver/rocsolver-functions.h [hunk 1]
```diff
@@ -24362,61 +24362,59 @@ ROCSOLVER_EXPORT rocblas_status rocsolver_dcsrrf_splitlu(rocblas_handle handle,
...
-    indM        pointer to rocblas_int. Array on the GPU of dimension nnzM.\n
+    indM        pointer to rocblas_int. Array on the GPU of dimension nnzM.
                 It contains the column indices of the non-zero elements of M. Indices are
                 sorted by row and by column within each row.
     @param[in]
-    valM        pointer to type. Array on the GPU of dimension nnzM.\n
-                The values of the non-zero elements of M. The strictly upper triangular entries are
-				not referenced when working in Cholesky mode.
+    valM        pointer to type. Array on the GPU of dimension nnzM.
+                The values of the non-zero elements of M.
```
Comment (tfalders) [inline library/include/rocsolver/rocsolver-functions.h]: This should be changed back to what is currently in develop

File: library/include/rocsolver/rocsolver-functions.h [hunk 2]
```diff
@@ -24362,61 +24362,59 @@ ROCSOLVER_EXPORT rocblas_status rocsolver_dcsrrf_splitlu(rocblas_handle handle,
...
-    indT        pointer to rocblas_int. Array on the GPU of dimension nnzT.\n
+    indT        pointer to rocblas_int. Array on the GPU of dimension nnzT.
                 It contains the column indices of the non-zero elements of T. Indices are
                 sorted by row and by column within each row.
     @param[in]
-    valT        pointer to type. Array on the GPU of dimension nnzT.\n
-                The values of the non-zero elements of T. The strictly upper triangular entries are
-                not referenced when working in Cholesky mode.
+    valT        pointer to type. Array on the GPU of dimension nnzT.
+                The values of the non-zero elements of T.
```
Comment (tfalders) [inline library/include/rocsolver/rocsolver-functions.h]: This too

File: library/include/rocsolver/rocsolver-functions.h [hunk 3]
```diff
@@ -24689,45 +24687,44 @@ ROCSOLVER_EXPORT rocblas_status rocsolver_dcsrrf_refactchol(rocblas_handle handl
...
```
Comment (tfalders) [inline library/include/rocsolver/rocsolver-functions.h]: This too

**Review by tfalders — COMMENTED** [review]
A few more changes to be made. Should be good after these.

**Review by tfalders — APPROVED** [review]
Looks good! Nice job

**Review by jzuniga-amd — APPROVED** [review]
LGTM

**Comment by cgmb** [comment]
There is a format check failure in `./library/include/rocsolver/rocsolver-functions.h`.

**Comment by tfalders** [comment]
If there are no objections, I'll force the merge once the readthedocs build and static analysis pass. Functional tests passed previously (and shouldn't be affected by this PR anyway).

**Pull request merged**